src/Pdk/Hooks/PdkProductSettingsHooks.php

```diff
@@ -155,8 +155,7 @@ public function savePdkProduct(array $post, int $productId): void
      */
     private function getProductSettingsKeys(array $post, ?int $productId = null): array
     {
-        $appInfo = Pdk::getAppInfo();
-        $postKey = $productId ? "$appInfo->name-childProductSettings--$productId-" : "$appInfo->name-";
+        $postKey = $productId ? "myparcelnl-childProductSettings--$productId-" : "myparcelnl-";
```

Comment:
waarom is dit veranderd? dit maakt geen verschil, en `$appInfo->name` heeft de voorkeur